Three/include/CGAL/Three/Scene_draw_interface.h

```diff
@@ -60,6 +60,8 @@ class Scene_draw_interface {
    * \param y the Y coordinate of theTextItem's position.
    * \param z the Z coordinate of theTextItem's position.
    * \param viewer the viewer used to display the Scene.
+   * \param scaler a vector indicating the scaling factors to apply to the scene when displaying it.
+   * It can be useful when a scene is very large along one of it's coordinates, making it hard to visualize it.
    * \return true if the TextItem is visible. */
   virtual bool  testDisplayId(double x, double y, double z, CGAL::Three::Viewer_interface* viewer, const QVector3D& scaler) = 0;
```

Comment:
Actually, I do not understand why the scene needs to know the scaling factor of the viewer. The scene should be agnostic to what happens in the viewer.